pegasus/sites.v3/code.org/public/hourofcode/artist.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Teacher Resources - Artist
+nav: hoc_teacher_resources_nav
+---
+
+# Artist: An Hour of Code Tutorial
+
+<center><iframe width="600" height="337" src="https://www.youtube.com/embed/MUavAHn-QUg" frameborder="0" allowfullscreen></iframe></center>
+
+[col-50]
+
+## Teacher Resources
+
+### Extension Activities:
+
+- [Beyond an Hour of Code](http://code.org/learn/beyond)
+- [Computer Science Fundamentals: Course 2](https://studio.code.org/s/course2)
+
+### Other Teacher Resources:
+
+- [Hour of Code: How-To Guide for Educators](https://hourofcode.com/us/en/resources/how-to)
+- [Best Practices from Successful Educators](http://www.slideshare.net/TeachCode/hour-of-code-best-practices-for-successful-educators-51273466)
+- Live Event: [An Educator's Guide to the Hour of Code](https://www.eventbrite.com/e/an-educators-guide-to-the-hour-of-code-tickets-17987415845)
+
+[/col-50]
+
+[col-50]
+
+## Overview
+
+Draw cool pictures and designs with the Artist!
+
+**Recommended Age Range:** 4-104
+
+**Technology Requirements:** Modern browsers + tablets
+
+<a href="http://studio.code.org/s/artist/reset"><button>Try the tutorial</button></a>
```

Comment:
I'll work on updating Melissa's pages w/ video partial & CDO links in a separate PR.